Cargo.toml

```diff
@@ -5,21 +5,21 @@ authors = ["aravk <aravkasi@gmail.com>"]
 edition = "2021"
 
 [dependencies]
-dotenv = "0.15"
```

Comment:
I think reading env vars from .env is still desirable functionality

Reply:
Rocket doesn't pick up the env vars from a .env file. You can have the same behavior with `Rocket.toml` (albeit with slightly more complicated configuration) including custom config variables. You can also pick these variables up from the environment, although changes are needed to the names.

Reply:
How can you have the same behavior with `Rocket.toml`? The whole point of the `.env` file is that you can gitignore the environment variables - how would you do that with `Rocket.toml`?

Reply:
You'd override the Rocket.toml config with environmental variables in production. Or have a separate, secret Rocket.toml using render and override the config location with an environmental variable.

Reply:
So should the entire `Rocket.toml` file be gitignored? According to your first suggestion, how would I specify sensitive information in development?